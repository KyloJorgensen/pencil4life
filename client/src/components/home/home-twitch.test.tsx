'use strict';

import * as React from 'react';
import { ShallowWrapper, shallow } from 'enzyme';
import TwitchIFrame from './home-twitch';

let wrapper: ShallowWrapper;

describe('Render TwitchIFrame', () => {
    it('should match the snaphot', () => {
        wrapper = shallow(
            <TwitchIFrame/>
        );
        expect(wrapper).toMatchSnapshot();
    });
});
