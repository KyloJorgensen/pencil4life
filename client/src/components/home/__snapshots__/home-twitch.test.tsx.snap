// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render TwitchIFrame should match the snaphot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <TwitchIframe />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <iframe
        allowFullScreen={true}
        frameBorder="0"
        onLoad={[Function]}
        scrolling="no"
        src="https://player.twitch.tv/?channel=pencil4life"
        style={
          Object {
            "height": "auto",
            "width": "auto",
          }
        }
      />,
      "className": "twitch-iframe-wrapper col-md-8 p-0",
    },
    "ref": Object {
      "current": null,
    },
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "allowFullScreen": true,
        "frameBorder": "0",
        "onLoad": [Function],
        "scrolling": "no",
        "src": "https://player.twitch.tv/?channel=pencil4life",
        "style": Object {
          "height": "auto",
          "width": "auto",
        },
      },
      "ref": Object {
        "current": null,
      },
      "rendered": null,
      "type": "iframe",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <iframe
          allowFullScreen={true}
          frameBorder="0"
          onLoad={[Function]}
          scrolling="no"
          src="https://player.twitch.tv/?channel=pencil4life"
          style={
            Object {
              "height": "auto",
              "width": "auto",
            }
          }
        />,
        "className": "twitch-iframe-wrapper col-md-8 p-0",
      },
      "ref": Object {
        "current": null,
      },
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "allowFullScreen": true,
          "frameBorder": "0",
          "onLoad": [Function],
          "scrolling": "no",
          "src": "https://player.twitch.tv/?channel=pencil4life",
          "style": Object {
            "height": "auto",
            "width": "auto",
          },
        },
        "ref": Object {
          "current": null,
        },
        "rendered": null,
        "type": "iframe",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
